### PR TITLE
fixes event documentation

### PR DESCRIPTION
**Pull request by ctaintor**
I noticed there were a few problems with the event documentation, so I updated it to be more complete/correct.

### CODE REVIEW

**Comment by obfuscurity** [comment]
Your "fix" describes the old behavior that was changed in https://github.com/graphite-project/graphite-web/pull/1627. I assume you're using a 0.9 release.

The URL fix is correct, would you like to rework the patch for that only?

**Comment by ctaintor** [comment]
Thanks for the info - I suspected as much but couldn't find the release notes (although I admit I didn't try that hard...).

I updated the PR

File: docs/events.rst [hunk 1]
```diff
@@ -74,7 +78,9 @@ It's also possible to dump the raw events using the API.
        }
     ]
 
-The ``set`` parameter accepts an optional ``union`` or ``intersection`` argument to determine the behavior for filtering sets of tags (i.e. inclusive or exclusive). By default, Graphite uses a "lazy union" that will return any matching events for a given tag in a list of tags. This behavior is not intuitive and will therefore be deprecated in a future release.
+The ``set`` parameter accepts an optional ``union`` or ``intersection`` argument to determine the behavior for filtering sets of tags (i.e. inclusive or exclusive). By default, Graphite uses a "lazy union" that will return any matching events for a given tag in a list of tags. This behavior is not intuitive and will therefore be deprecated in a future release. When matching using multiple tags, separate the tags with a space.
+
+*Note*: Prior to 0.10.0, the value of ``tags`` will be a string, with multiple tags being separated by a space.
```
Comment (ctaintor) [inline docs/events.rst]: good point. In my case, I was talking about the output of get_data changing. I agree it's confusing so added what you suggested and clarified what I wrote.

Comment (obfuscurity) [inline docs/events.rst]: Ok, this still needs to be cleaned up. I really dislike that sentence being slapped onto the `set` paragraph, it doesn't belong there. And I don't think we need that *Note* since it's referring to the output. Nowhere else do we explicitly document the format of each key/value output. As long as we have an accurate example (which we do, [here](https://github.com/graphite-project/graphite-web/blame/master/docs/events.rst#L56-L75)), I don't see the point.

TL;DR please remove the changes in this snippet.

Comment (ctaintor) [inline docs/events.rst]: Sorry - I had meant to remove that sentence. I'll remove it. (the "When matching...")

About the output, I only have it there because it changed - in 0.9, the output doesn't match the example. It seemed worth noting. If you feel that has no value, I will remove it as well.

**Comment by obfuscurity** [comment]
Thanks!
